app/SendAmount.tsx

```diff
@@ -41,31 +44,48 @@ export default function SendAmount() {
   const { isRequest } = params;
 
   const onPressSend = async () => {
-    dispatch(setIsUserLoading(true));
     if (!chosenContact?.phoneNumbers) return;
 
     // phone number for recipient in send flow and 'purchaser' (aka requestee) in request flow
     const phoneNumber = chosenContact?.phoneNumbers[0].number;
 
     if (!phoneNumber) return;
+    if (currentUser?.accountBalance <= 0) {
+      throw new Error('not enough');
+    }
+
+    if (amount > currentUser?.accountBalance) {
+      dispatch(setModalText('Not enough funds'));
```

Comment:
The text 'Not enough funds' need to be in available in all translations. Write here as:
```suggestion
      dispatch(setModalText(i18n.t('sendamount_not_enough_funds'));
```
and add text to:
- app\translationService\en.ts
- app\translationService\he.ts
- app\translationService\ar.ts